src/Pimple/PimpleContainerAdapter.php

```diff
@@ -48,6 +48,8 @@ public function addServiceConfig(ServiceConfig $config)
     }
 
     /**
+     * @param InflectorConfig $config
```

Comment:
I use this as a rule of thumb:

There is no need to add a docblock if 

* a method doesn't return anything and doesn't accept arguments
* a method doesn't return anything and all arguments are _sufficiently_ type-hinted
* a method returns something, but we can use return type declarations
* a method doesn't throw any exceptions
* a method implements an interface and the method in the interface or abstract class has a _sufficient_ docblock

For example

```php
private function foo()

public function __construct(Foo $foo, array $options = [])

public static function from(Foo $foo) : Bar;
```

A docblock should be added if 

* not all arguments can be _sufficiently_ type-hinted
* a method throws exceptions
* a method returns something, but return type declarations can not be used

For example

```php
/**
 * @param Foo $foo
 * @param Bar[] $bars
 * 
 * @throws \RuntimeException
 *
 * @return Baz[]
 */
public function baz(Foo $foo, array $bars)
{
    if ( ) {
        throw new \RuntimeException();
    }
 
    return array_map(function (Bar $bar) {
        return new Baz($bar);
    }, $bars);
}
```

The idea is to aid developers so they can see

* what types are expected for arguments
* what types are returned
* what exceptions are thrown (and need to be caught)

💁 When there is a reason to add a docblock, then it should be a _full_ docblock.

What do you think?